lib/myfinance/resources/entity.rb

```diff
@@ -0,0 +1,41 @@
+module Myfinance
+  module Resources
+    #
+    # A wrapper to Myfinance entities API
+    #
+    # [API]
+    #   Documentation: https://app.myfinance.com.br/docs/api/entities
+    #
+    class Entity < Base
+      attr_reader :id, :name, :account_id, :charging_uuid, :created_at, :default_in_menu, :deleted_at, :federation_subscription_number, :imported_from_sync_at, :updated_at
+
+      #
+      # List all entities of the user
+      #
+      # [API]
+      #   Method: <tt>GET /entities</tt>
+      #
+      #   Documentation: https://app.myfinance.com.br/docs/api/entities#get_index
+      #
+      def self.find_all
+        client.get('/entities', body: {}) do |response|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.